flink-runtime-web/web-dashboard/src/app/pages/job/exceptions/job-exceptions.component.html

```diff
@@ -40,23 +40,25 @@
         <ng-container *ngFor="let exception of listOfException;trackBy:trackExceptionBy;">
           <tr>
             <td nzShowExpand [(nzExpand)]="exception.expand"></td>
-            <td>{{exception.timestamp | date:'yyyy-MM-dd HH:mm:ss'}}</td>
-            <td><div class="name">{{exception.exceptionName}}</div></td>
+            <td>{{exception.selected.timestamp | date:'yyyy-MM-dd HH:mm:ss'}}</td>
+            <td><div class="name">{{exception.selected.exceptionName}}</div></td>
             <td>
-              <div class="name">
-                  {{exception.taskName || "(global failure)"}}
-              </div>
+              <nz-select style="width: 300px;" [nzDisabled]="exception.values.length == 1" nzShowSearch [(ngModel)]="exception.selected">
```

Comment:
I am a bit skeptical of using `nzDisabled` in the case of a single exception, but I will have to try and see if I have another suggestion.

Reply:
I have a slight preference to just not disable it in the case of a single option, but it's fine either way!